apps/entropy-tester/src/index.ts

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env node
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import YAML from "yaml";
+import fs from "fs";
+import pino, { Logger } from "pino";
+import { DefaultStore } from "@pythnetwork/contract-manager/node/store";
+import { EvmEntropyContract } from "@pythnetwork/contract-manager/core/contracts/evm";
+import {
+  PrivateKey,
+  toPrivateKey,
+} from "@pythnetwork/contract-manager/core/base";
+
+type DurationSeconds = number;
+type LoadedConfig = {
+  contract: EvmEntropyContract;
+  interval: DurationSeconds;
+};
+
+function timeToSeconds(timeStr: string): number {
+  const match = timeStr.match(/^(\d+)([hms])$/i);
+  if (!match)
+    throw new Error("Invalid format. Use formats like '6h', '15m', or '30s'.");
+
+  const value = parseInt(match[1], 10);
+  const unit = match[2].toLowerCase();
+
+  switch (unit) {
+    case "h":
+      return value * 3600;
+    case "m":
+      return value * 60;
+    case "s":
+      return value;
+    default:
+      throw new Error("Unsupported time unit.");
+  }
+}
+
+function loadConfig(configPath: string): LoadedConfig[] {
+  const configs = YAML.parse(fs.readFileSync(configPath, "utf-8"));
+  const loadedConfigs = [];
+  for (const config of configs) {
+    const interval = timeToSeconds(config["interval"]);
+    const contracts = Object.values(DefaultStore.entropy_contracts).filter(
+      (contract) => contract.chain.getId() == config["chain-id"],
+    );
+    if (contracts.length === 0) {
+      throw new Error(
+        `Can not find the contract for chain ${config["chain-id"]}, check contract manager store.`,
+      );
+    }
+    if (contracts.length > 1) {
+      throw new Error(
+        `Multiple contracts found for chain ${config["chain-id"]}, check contract manager store.`,
+      );
+    }
+    loadedConfigs.push({ contract: contracts[0], interval });
```

Comment:
Doing:

```ts
const foo = [];
for (const bar of bars) {
  foo.push(getSomeValueFrom(bar));
}
```

is just a procedural equivalent to:

```ts
const foo = bars.map(bar => getSomeValueFrom(bar))
```

I generally recommend getting in the habit of the declarative map over the procedural loop